lib/providers/gcm.js

```diff
@@ -68,7 +68,7 @@ GcmProvider.prototype._createMessage = function(notification) {
   });
 
   Object.keys(notification).forEach(function (key) {
-    if (notification[key] != null) {
+    if (notification[key]) {
```

Comment:
Good catch, I'll create a specific simple test as mentioned above to check that `null` and `undefined` are ignored while other values aren't.